.github/workflows/deploy.yml

```diff
@@ -118,16 +118,13 @@ jobs:
     name: Build and deploy Android HybridApp
     needs: prep
     runs-on: ubuntu-latest-xl
-    defaults:
-      run:
-        working-directory: Mobile-Expensify/react-native
+    env:
+      RUBYOPT: '-rostruct'
```

Comment:
We removed this old workaround recently. Unless you re-added this for a specific reason can you remove it please? I assume it came from a `main` merge